### DISCUSSION TITLE
Mark incoming messages as spam or newsletter / ignore

### DISCUSSION THREAD

**Discussion by mrusteberg** [chatwoot/chatwoot#3967] [Feature Requests]
Certain senders continually send spam or newsletters. It should be possible to ignore (newsletters) or block (spam) messages based on sender details. If e-mail channel, allow to block whole domain or just a sender

![image](https://user-images.githubusercontent.com/54262281/153591973-c42108e9-07cb-4c67-b483-a750c1cf26dd.png)

**Comment by sojan-official** [comment]
tracking this in https://github.com/chatwoot/chatwoot/issues/3996
